index.js

```diff
@@ -34,6 +34,8 @@ function username(cwd, verbose) {
 
   if (path.length && path.charAt(0) === '/') {
     path = path.slice(1);
+  } else if (path.length && path.match(/git@.*\:/)) {
+    path = path.split(':')[1];
```

Comment:
Thanks for the pr! Instead of doing `.match` then splitting, maybe we should change the regex a bit and use the object returned by `.match` (or preferably `.exec` since `.match` just wraps exec under the hood)?